Project.toml

```
@@ -56,7 +56,7 @@ Distances = "0.10"
 DocStringExtensions = "0.8, 0.9"
 Enzyme = "0.13.14"
 FFTW = "1"
-GPUArrays = "10.3"
+GPUArrays = "11.2.2"
```

If you don't need to make changes here to adapt to other breaking changes in GPUArrays maybe you can allow them both?  Like
```suggestion
GPUArrays = "10.3, 11.2.2"
```

---

good point